app/models/setting/general.rb

```diff
@@ -17,7 +17,9 @@ def self.default_settings
       self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
       self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout')),
       self.set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text')),
-      self.set('host_power_status', N_("Show power status on host index page. This feature calls to compute resource providers which may lead to decreased performance on host listing page."), true, N_('Show host power status'))
+      self.set('host_power_status', N_("Show power status on host index page. This feature calls to compute resource providers which may lead to decreased performance on host listing page."), true, N_('Show host power status')),
+      self.set('http_proxy', N_('Set a HTTP(S) proxy for all outgoing connections'), nil, N_('HTTP(S) Proxy')),
```

Comment:
I think it would deserve more detailed description, e.g. communication with smart proxy seems to be unaffected. Also an example (hostname:port) might be useful. The same for the setting below.